test/test_common/logging.h

```diff
@@ -78,11 +83,13 @@ using ExpectedLogMessages = std::vector<StringPair>;
 //  Does NOT contain:
 //    'warning', 'Too many sendDiscoveryRequest calls for baz’
 //    'warning', 'Too man sendDiscoveryRequest calls for foo'
-#define EXPECT_LOG_CONTAINS_ALL_OF(expected_messages, stmt)                                        \
+#define EXPECT_LOG_CONTAINS_ALL_OF_ESCAPED(expected_messages, stmt, escaped)                       \
```

Comment:
it's weird to have this named ESCAPED when you have to then also pass a bool.

How about calling this one EXPECT_LOG_CONTAINS_ALL_OF_HELPER

and then having EXPECT_LOG_CONTAINS_ALL_OF_ESCAPED just take 2 args and pass true to the HELPER?

Reply:
Right, the reason I did this is because a lot of tests use `EXPECT_LOG_CONTAINS_ALL_OF` and I didn't want to rename it in all the files. For now, i'll change `EXPECT_LOG_CONTAINS_ALL_OF_ESCAPED` to be `EXPECT_LOG_CONTAINS_ALL_OF_HELPER`, let me know if there is a better solution.